### PR TITLE
[EASI-2606] Updating Truss package

### PR DESCRIPTION
**Pull request by garyjzhao**
# EASI-2606

## Changes and Description

- Update Truss package to 6.0.0
- Update dockerfile.frontend to use node 16.20.0
- update `.nvmrc` to use 16.20.0
- Updated `Alert` component to have headingLevel prop (default heading level has been deprecated, so it is now necessary to write it out)
- Update the way Icon is used in app (<Icon.{IconName} /> instead of <Icon{IconName} />)
- Updated SumamryBox to have sub-components  (`SummaryBoxHeading` and `SummaryBoxContent`)
- Replaced deprecated `Dropdown` component with `Select` component 

## How to test this change
1. `yarn install` and bring up environment
2. go through the app and ensure pages load and still match figma 
3. Check lint, test, and cypress tests pass

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: src/index.scss
Comment (garyjzhao) [inline src/index.scss:60]: Removing this as per my discussion with @patrickseguraoddball, because its inclusion is overwriting all the overwrites we implemented in `_settings.scss` file. 
Some things in particular were like max-width to the grid containers were smaller than what we currently have, a blue header to all tables, etc.

File: src/views/ModelPlan/CRTDL/CRTDLs/__snapshots__/index.test.tsx.snap
Comment (patrickseguraoddball) [inline src/views/ModelPlan/CRTDL/CRTDLs/__snapshots__/index.test.tsx.snap:277]: These snapshots are duplicating and failing to render the data.  It doesnt seem too related to this PR and hopefully addressed in the unit test PR.  I would wait for the other PR to be merged and recheck if this get updated correctly

File: src/views/ModelPlan/Status/__snapshots__/index.test.tsx.snap
Comment (patrickseguraoddball) [inline src/views/ModelPlan/Status/__snapshots__/index.test.tsx.snap:203]: Quite a few of these snapshots are taking additional snapshots instead of updating existing ones.  We should look into this to figure out why this is happening in all the tests.  May be addressed in the other PR as well but should check back

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
Everything looks and works great/as expected.  There's some weirdness with the snapshots that might be addressed by the other open PR.  We should check back on these snapshots after thats merged into this branch from main

**Comment by StevenWadeOddball** [comment]
Thanks Gary, I pulled this locally and looked for anything out of the ordinary. It looks good to me so far!

**Review by patrickseguraoddball — APPROVED** [review]
LG!

**Pull request merged**
